src/containers/BooksForm.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { CREATE_BOOK } from '../actions';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleCategory = this.handleCategory.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  handleCategory(event) {
+    this.setState({
+      category: event.target.value,
+    });
+  }
```

Comment:
- Good job on implementing these two, you can combine these two methods into one (`handleChange`) as suggested in the requirements, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleChange.png)

**hint**
you can do this by using `event.target.name` and `event.target.value`